File: trisq.java
import java.util.*;

class trisq{
    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        int t=in.nextInt();
        while(t-->0){
            int b=in.nextInt();
            int c=(b/2)-1;
            int res=0;
            for(int i=1;i<=c;i++){
                res+=i;
            }
            System.out.println(res);
        }
    }
}

File: trisq.py
t=int(input())
for i in range(t):
    b=int(input())
    #print(int(b/2))
    c=(int(b/2)-1)
    #print(c)
    res=0
    for j in range(c+1):
        res+=j
    
    print(res)
